foundry/gui/HeaderEditor.py

```diff
@@ -80,6 +80,7 @@ def __init__(self, parent: Optional[QWindow], level_ref: LevelRef):
         super(HeaderEditor, self).__init__(parent, "Level Header Editor")
 
         self.level: Level = level_ref.level
+        assert self.level is not None
```

Comment:
This is added to ensure that the type hints are nice and that the level is not None, which is a possibility denoted by level_ref currently.